### PR TITLE
Fix y-coordinate typo in createExitData

### PR DESCRIPTION
**Pull request by Pirijuamps**
Commit [7488171](https://github.com/le-doux/bitsy/commit/74881711335ef6b92238d53f458b6fa6ac4ecf8c) in part addressed issue [#171](https://github.com/le-doux/bitsy/issues/171). In this fix, function ‘createExitData’ was made with a typo and exits are now being created with both coordinates pointing to x. 
 
 This PR only corrects the typo on createExitData so y-coordinate is assigned properly.

### CODE REVIEW

**Comment by frojo** [comment]
would it be okay to ask for another reviewer to get this fix pushed soon? it seems adam is busy with other things at the moment

i'm teaching bitsy to a group of students next week and it would be nice if exits and endings worked :)

also totally understand if people are busy! thank you for making bitsy exist and work all these years

**Comment by seleb** [comment]
> would it be okay to ask for another reviewer to get this fix pushed soon? it seems adam is busy with other things at the moment
> 
> i'm teaching bitsy to a group of students next week and it would be nice if exits and endings worked :)
> 
> also totally understand if people are busy! thank you for making bitsy exist and work all these years

hey there, the fix is simple but merging the PR requires adam's approval. if you need access to a working version in the meantime though, you have a couple options:

1. you could download a copy of the editor from before the bug was introduced (e.g. from 36d9a39a47976b3ec67bbef82e7d83ee8ba14824, go to "browse files" and then "code" > "download zip")
2. you could download a copy of the current editor and apply this fix manually

**Comment by frojo** [comment]
ah sorry, didn't realize adam was the sole approver

ok thank you seleb! I did go ahead and make a temporary fork

**Review by le-doux — APPROVED** [review]
oops D:

(thanks for finding this!)

**Comment by le-doux** [comment]
thanks for working on this y'all! (and thanks for your patience everyone! i've had a lot of personal stuff going on)

**Pull request merged**
